libbeat/docs/shared-kerberos-config.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[configuration-kerberos]]
```

Comment:
It looks like you haven't included this file anywhere. It's not being built, which is why the link [here](https://github.com/elastic/beats/pull/17927/files#diff-1eea36cd1513c862a97a56d38522dd58R684) fails.

Reply:
Thank you for your help!